components/frontend/eslint.config.mjs

```diff
@@ -0,0 +1,63 @@
+import { fixupConfigRules } from "@eslint/compat"
+import pluginJs from "@eslint/js"
+import pluginJest from "eslint-plugin-jest"
+import pluginNode from "eslint-plugin-n"
+import pluginPrettierConfigRecommended from "eslint-plugin-prettier/recommended"
+import pluginPromise from "eslint-plugin-promise"
+import pluginReactJSXRuntime from "eslint-plugin-react/configs/jsx-runtime.js"
+import pluginReactConfigRecommended from "eslint-plugin-react/configs/recommended.js"
+import pluginSimpleImportSort from "eslint-plugin-simple-import-sort"
+import globals from "globals"
+
+export default [
+    pluginJs.configs.recommended,
+    pluginJest.configs["flat/recommended"],
+    pluginNode.configs["flat/recommended-module"],
+    ...fixupConfigRules(pluginReactConfigRecommended),
+    ...fixupConfigRules(pluginReactJSXRuntime),
+    pluginPrettierConfigRecommended,
+    {
+        plugins: {
+            "simple-import-sort": pluginSimpleImportSort,
+            promise: pluginPromise,
+        },
+        languageOptions: {
+            globals: { ...globals.browser, ...globals.node },
+            parserOptions: {
+                ecmaFeatures: {
+                    jsx: true,
+                },
+            },
+        },
+        rules: {
+            "no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
+            "n/no-unsupported-features/node-builtins": "off",
+            "promise/always-return": "error",
+            "promise/no-return-wrap": "error",
+            "promise/param-names": "error",
+            "promise/catch-or-return": ["error", { allowFinally: true }],
+            "promise/no-native": "off",
```

Comment:
What's that?

Reply:
Not relevant for us AFAICT: https://github.com/eslint-community/eslint-plugin-promise/blob/main/docs/rules/no-native.md